tests/e2e/fixtures.ts

```
@@ -0,0 +1,65 @@
+  test as base,
+  type Worker,
+} from '@playwright/test'
+
+const here = dirname(fileURLToPath(import.meta.url))
+const extensionPath = resolve(here, '../../.output/chrome-mv3')
+const chrome =
+  process.env.PLAYWRIGHT_CHROME ??
+  '/etc/profiles/per-user/yb/bin/google-chrome'
+
+export const test = base.extend<{ context: BrowserContext }>({
+  context: async ({}, use) => {
```

Biome (2.4.16)

[error] Unexpected empty object pattern.

(lint/correctness/noEmptyPattern)